osgtest/library/files.py

```diff
@@ -144,12 +144,9 @@ def replace(path, old_line, new_line, owner=None, backup=True):
     write(path, lines_to_write, owner, backup)
 
 def replace_regexpr(path, regexp, new_line, owner=None, backup=True):
-    """Replace an old line that matches regexpr with a new line in given path.                                                     
-    The 'owner' and 'backup' arguments are passed to write().                                                                                  
+    """Replace an old line that matches regexpr with a new line in given path.
+    The 'owner' and 'backup' arguments are passed to write().
     """
-    match = re.search(regexp, new_line)
-    if not match:
-       raise ValueError("The new_line '%s' does not match reg expr '%s'" % (new_line, regexp)) 
     lines_to_write = []
```

Comment:
Cool, github has an option to view files via `git blame`: https://github.com/opensciencegrid/osg-test/blame/master/osgtest/library/files.py#L150-L152

I didn't see @efajardo online at all to ask him about it so I just went with the nuclear option and blew it away because I couldn't think of any good reason to keep it.